### DISCUSSION TITLE
Two factor auth with harware security keys don't work in Firefox, Chrome

### DISCUSSION THREAD

**Discussion by silasmarques** [community/community#37905] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello good afternoon.
I am not able to register 2-factor authentication. I do the Scan, and when typing the code does not recognize. display message

Two-factor code verification failed. Please try again.

Thanks

**Comment by silasmarques** [comment, marked as answer]
Resolved

**Reply by hbetabessi** [reply]
how ? im facing the same issue

**Reply by mecchmatProjects** [reply]
HOW????????????????????????????????
